lib/puppet/type/ini_setting.rb

```diff
@@ -5,6 +7,19 @@
     defaultto :present
   end
 
+  def munge_boolean_md5(value)
+    case value
+    when true, :true, 'true', :yes, 'yes'
+      true
```

Comment:
Change looks pretty good, take a look here though https://github.com/puppetlabs/puppet/blob/master/lib/puppet/type/macauthorization.rb#L20-L29 you'll notice it should return :true or :false not true or false as the insync? seems to have an issue if it's not a symbol(I don't know why, just tested it and found it to be the correct method)